File: calsalario.kt
fun main(args: Array<String>) {
  // Ler o valor bruto do salário e o adicional dos benefícios
  val salarioBruto = readLine()?.toDouble() ?: 0.0
  val adicionalBeneficios = readLine()?.toDouble() ?: 0.0

  // Calcular o percentual de imposto
  val percentualImposto = when (salarioBruto) {
    in 0.0..1100.0 -> 0.05
    in 1100.1..2500.0 -> 0.10
    else -> 0.15
  }

  // Calcular o salário a ser transferido
  val salarioTransferido = salarioBruto - (salarioBruto * percentualImposto) + adicionalBeneficios

  // Imprimir o salário a ser transferido
  println("Salário a ser transferido: R$${salarioTransferido:.2f}")
}

File: calsalario.cs
using System;

class DesafioSalario
{
    static void Main(string[] args)
    {
        // Ler o valor bruto do salário
        double salarioBruto = double.Parse(Console.ReadLine());

        // Ler o adicional dos benefícios
        double adicionalBeneficios = double.Parse(Console.ReadLine());

        // Calcular o percentual de imposto
        double percentualImposto;
        if (salarioBruto <= 1100.00)
        {
            percentualImposto = 0.05;
        }
        else if (salarioBruto <= 2500.00)
        {
            percentualImposto = 0.10;
        }
        else
        {
            percentualImposto = 0.15;
        }

        // Calcular o salário a ser transferido
        double salarioTransferido = salarioBruto - (salarioBruto * percentualImposto) + adicionalBeneficios;

        // Imprimir o salário a ser transferido
        Console.WriteLine(salarioTransferido);
    }
}

